### PR TITLE
Update set-up.qmd to improve clarity and readability

### PR DESCRIPTION
**Pull request by YaoxiangLi**

### CODE REVIEW

File: publish/set-up.qmd [hunk 1]
```diff
@@ -4,8 +4,7 @@ title: "Set up your own universe"
 
 ## Basic steps
 
-Creating a universe for your R packages can be done in a few steps described below.
-The further sections on this page give more details on each step.
+Setting up a universe for your R packages involves just a few steps, outlined below. Additional details for each step are provided in the sections that follow.
```
Comment (maelle) [inline publish/set-up.qmd:7]:
```suggestion
Setting up a universe for your R packages involves just a few steps, outlined below. 
Additional details for each step are provided in the sections that follow.
```

File: publish/set-up.qmd [hunk 2]
```diff
@@ -17,19 +16,18 @@ The further sections on this page give more details on each step.
 
 ## Creating your personal R-universe subdomain {#subdomain}
 
-In R-universe, every user has a private subdomain, where you control the content, and which does not interfere with other users. 
-The subdomains are tied to the GitHub account or organization with that name. 
+Each user gets a private subdomain on R-universe, where you control the content independently. Subdomains are tied to GitHub accounts or organizations with the same name.
```
Comment (maelle) [inline publish/set-up.qmd:19]:
```suggestion
Each user gets a private subdomain on R-universe, where you control the content independently. 
Subdomains are tied to GitHub accounts or organizations with the same name.
```

File: publish/set-up.qmd [hunk 3]
```diff
@@ -165,12 +163,10 @@ options(repos = c(
 install.packages('tinkr')
 ```
 
-To help package users, you can consider copying these instructions to your package README.md files. 
-And of course you can add an R-universe badge to the package README as well.
+Copy these instructions to your package `README.md` files to assist users. You can also add an R-universe badge to indicate the deployed version:
```
Comment (maelle) [inline publish/set-up.qmd:166]:
```suggestion
Copy these instructions to your package `README.md` files to assist users. 
You can also add an R-universe badge to indicate the deployed version:
```

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you!! :sparkles: 

I'd just like to add line breaks at the end of sentences. :smile_cat:

**Comment by YaoxiangLi** [comment]
> Thank you!! ✨
> 
> I'd just like to add line breaks at the end of sentences. 😸

Got it, thanks!

**Pull request merged**